src/index.html

```diff
@@ -11,8 +11,46 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__imac">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+        />
+      </div>
+      <h1 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__subtitle">Product code: 195434</p>
+      <div class="card__review">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <p class="card__review-label">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <span class="card__price-text">Price:</span>
+        <span class="card__price-price">$2,199</span>
+      </div>
+      <a
+        class="card__button"
+        href="#"
+      >
+        BUY
```

Comment:
The 'BUY' text should be wrapped in a 'span' or similar inline element with a BEM class for styling purposes. It's a good practice to separate structure from content.